src/towncrier/newsfragments/428.feature

```diff
@@ -0,0 +1 @@
+Ability to create fragments not tied to issues via a leading `+` in the filename (e.g. `+anything.feature`).
```

Comment:
Maybe a bit more explicit, and in the "tone" of a release note.

```suggestion
You can now create fragments that are not associated with issues. Start the name of the fragment with `+` (e.g. `+anything.feature`). The content will be included in the final release notes.
```

Reply:
Improved this.